tests/NuGet.Services.AzureSearch.Tests/HijackDocumentBuilderFacts.cs

```diff
@@ -395,12 +395,24 @@ public void SetsLicenseUrlToGalleryWhenPackageHasLicenseFile()
 
                 Assert.Equal(Data.GalleryLicenseUrl, document.LicenseUrl);
             }
+
+            [Fact]
+            public void SetsIconUrlToFlatContainerWhenPackageHasIconFile()
```

Comment:
This seem to only test the situation when we have both iconUrl and iconFile specified. How about other combinations?

Reply:
just `iconUrl` is already tested. I will add one for just `iconFile`